src/render/quantization.cpp

```diff
@@ -44,8 +45,11 @@ Palette* create_palette_from_sprite(
   const bool withAlpha,
   Palette* palette,
   TaskDelegate* delegate,
-  const bool newBlend)
+  const bool newBlend,
+  MapAlgorithm mappingAlgorithm)
 {
+
+  OctreeMap* map = new OctreeMap();
```

Comment:
Is this `map` always necessary? e.g. couldn't we create the map only when the algorithm is OCTREE?

There is a memory leak too, two memory leaks to be exact (see below).